integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1960,36 +1960,36 @@ msgid ""
 "Select this option to create an inactive user account and send an activation "
 "link per email to the user."
 msgstr ""
-"Wählen Sie diese Option, um ein inaktives Nutzerkonto zu erstellen und einen "
-"Link zu dessen Aktivierung per Email an den Nutzer zu senden."
+"Wählen Sie diese Option, um ein inaktives Konto zu erstellen und einen Link "
+"zu dessen Aktivierung per Email zu versenden."
 
 #: cms/forms/users/user_form.py:51
 msgid ""
 "This link allows the user to choose a password and activates the account "
 "after confirmation."
 msgstr ""
-"Dieser Link erlaubt es dem Nutzer ein Passwort auszuwählen und aktiviert das "
-"Konto nach Bestätigung."
+"Dieser Link erlaubt es Nutzer*innen ein Passwort auszuwählen und aktiviert "
```

Comment:
```suggestion
"Dieser Link erlaubt es Benutzer*innen ein Passwort auszuwählen und aktiviert "
```
I think we could replace `Nutzer*in` with `Benutzer*in` to be more consistent in our wording (or use "Konto" if we decide on that term for a user account).